src/qcodes/parameters/multi_channel_instrument_parameter.py

```diff
@@ -12,6 +13,7 @@
     from .parameter_base import ParamRawDataType
 
 InstrumentModuleType = TypeVar("InstrumentModuleType", bound="InstrumentModule")
+LOG = logging.getLogger(__name__)
```

Comment:
Since writing many of the QCODES modules we have learned that this probably should be private

```suggestion
_LOG = logging.getLogger(__name__)
```